docs/Getting Started.md

```diff
@@ -0,0 +1,113 @@
+## Getting Started
+
+Hokusai aims to provide a smooth CI workflow on a per-project basis.  As such, it is designed to be configured and run from a shell within a project's git repo.
+
+We will assume you have already installed Hokusai and run `hokusai configure`, as detailed in [Setup](#Setup).
+
+1) Set up Hokusai for a Rails project
+  ```bash
+  cd ./path/to/my/rails/project/git/repo
+  hokusai setup --aws-account-id 12345 --project-type ruby-rails
+  ```
+
+  Note: If you set the environment variable `AWS_ACCOUNT_ID` in your shell, you can omit the `--aws-account-id` option.
+
+  `hokusai setup` will create:
+    - A `Dockerfile` in your project root
+    - A configuration folder `./hokusai`.  This folder contains:
+      `./hokusai/config.yml` - contains Hokusai project configuration
+      `./hokusai/common.yml` - a Docker Compose YAML file imported by `development.yml` and `test.yml`
+      `./hokusai/development.yml`- a Docker Compose YAML file for the development environment
+      `./hokusai/test.yml` - a Docker Compose YAML file for the test environment
+      `./hokusai/staging.yml` - a Kubernetes YAML file for the staging environment
+      `./hokusai/production.yml` - a Kubernetes YAML file for the production environment
```

Comment:
Use a bulleted list or some kind of clearer formatting for this list. It's currently unreadable.